packages/arb-token-bridge-ui/src/util/rpc/getRpcUrl.ts

```diff
@@ -0,0 +1,49 @@
+import { ChainId } from '../../types/ChainId'
+
+import { getInfuraRpcUrl } from './infura'
+import { getAlchemyRpcUrl } from './alchemy'
+
+type RpcProvider = 'infura' | 'alchemy'
+
+export type ProductionChainId = Exclude<
+  ChainId,
+  ChainId.Local | ChainId.ArbitrumLocal | ChainId.L3Local
+>
+
+function getRpcProvider(): RpcProvider {
+  const rpcProviderFromEnv = process.env.NEXT_PUBLIC_RPC_PROVIDER
```

Comment:
We should add this to `.env.local.sample` and add comment about the default value there.